assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -94,8 +94,11 @@ che.workspace.terminal_linux_arm7=${che.home}/lib/linux_arm7/terminal
 
 che.workspace.exec_linux_amd64=${che.home}/lib/linux_amd64/exec
 
-# RAM default for each machine that has no RAM settings in environment.
-che.workspace.default_memory_mb=1024
+# RAM limit default for each machine that has no RAM settings in environment.
+che.workspace.default_memory_limit_mb=1024
+# RAM request default for each machine that has no explicit RAM settings in environment.
+# this amount will be allocated on workspace container creation
```

Comment:
Can you add a comment that infra implementation might not respect this property?